### PR TITLE
feat: update imprint page

### PR DESCRIPTION
**Pull request by YouQam**

### CODE REVIEW

File: src/app/public/components/impressum/impressum.component.html [hunk 1]
```diff
@@ -5,66 +5,57 @@
...
-				<br /><br />
 
-				<span style="font-weight: bold"> mycelia gGmbH </span>
+				<span class="tw-font-bold">Vertretungsberechtigt:</span>
+				<br />
+				Franziska Schmid und Julia Kleeberger
+				<br />
+				<span class="tw-block tw-mt-4"
+					>E-Mail:
+					<a href="mailto:post@mycelia.education">
```
Comment (jona159) [inline src/app/public/components/impressum/impressum.component.html:39]: Please indicate this link as well

File: src/app/public/components/impressum/impressum.component.html [hunk 2]
```diff
@@ -5,66 +5,57 @@
...
-					openSenseLab gGmbH
-				</span>
-				<br />
-				Von-Steuben-Str. 21
-				<br />
-				48143 Münster
-				<br /><br />
+			<p style="font-size: 1.1rem; color: black; line-height: 1.55; margin: 2em 0 1.3em">
+				Open Educational Badges ist ein Gemeinschaftsprojekt von
+				<a href="https://matrix-gruppe.de/matrix-gemeinnuetzig/"> <u>matrix gGmbh</u> </a>,
```
Comment (jona159) [inline src/app/public/components/impressum/impressum.component.html:10]: Can you make it more clear visually that this is a link? I think we use these classes for links: ```class='tw-underline tw-text-[#1400FF]'``` Also, i guess it should probably open a new tab

**Review by jona159 — CHANGES_REQUESTED** [review]

File: src/app/public/components/impressum/impressum.component.html (hunk 2 above)
Comment (jona159) [reply src/app/public/components/impressum/impressum.component.html:10]: I think it would be good to open the links here in a new tab, but i will approve

**Review by jona159 — APPROVED** [review]

File: src/app/public/components/impressum/impressum.component.html [hunk 3]
```diff
@@ -5,66 +5,57 @@
...
-				Eylauer Str. 14
+				Amtsgericht Charlottenburg (Berlin) - HRB 251293 B
 				<br />
-				10965 Berlin
+				Steuer-Nr: 27/614/05396
+
 				<br />
-				Deutschland
+				<br />
+				Inhaltlich verantwortlich nach § 18 MStV ist Franziska Schmidt, erreichbar über die Dienstanbieterin.
```
Comment (zven) [inline src/app/public/components/impressum/impressum.component.html:54]: There was a typo in the original text: It's `Schmid` not `Schmidt`

**Pull request merged**